utils/networks.ts

```diff
@@ -18,34 +23,124 @@ export const networks = {
     railgunNetworkName: NetworkName.Ethereum,
     chainId: mainnet.id,
     wethAddress: "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
+    evmGasType: EVMGasType.Type2,
+    baseToken: {
+      symbol: "ETH",
+      name: "Ether",
+    },
+    fallbackProviders: {
+      chainId: mainnet.id,
+      providers: [
+        {
+          provider: "https://cloudflare-eth.com/",
+          priority: 1,
+          weight: 1,
+        },
+        {
+          provider: "https://rpc.ankr.com/eth",
+          priority: 2,
+          weight: 1,
+        },
+      ],
+    },
   },
   [goerli.id]: {
     blockExplorerUrl: "https://goerli.etherscan.io/",
     tokenListUrl: "https://gateway.ipfs.io/ipns/tokens.uniswap.org",
     railgunNetworkName: NetworkName.EthereumGoerli,
     chainId: goerli.id,
     wethAddress: "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
+    evmGasType: EVMGasType.Type2,
+    baseToken: {
+      symbol: "GoerliETH",
+      name: "Goerli Ether",
```

Comment:
Should be the same as mainnet: symbol of `ETH` and name of `Ether`

Reply:
Goodbye, Goerli ETH 7cd7d57